src/Compilers/VisualBasic/Portable/Compilation/SyntaxTreeSemanticModel.vb

```diff
@@ -29,7 +29,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Private ReadOnly _ignoresAccessibility As Boolean
 
         ' maps from a higher-level binder to an appropriate SemanticModel for the construct (such as a method, or initializer).
-        Private ReadOnly _semanticModelCache As New ConcurrentDictionary(Of Tuple(Of Binder, Boolean), MemberSemanticModel)()
+        Private ReadOnly _semanticModelCache As New ConcurrentDictionary(Of (binder As Binder, ignoresAccessibility As Boolean), MemberSemanticModel)()
```

Comment:
>(binder As Binder, ignoresAccessibility As Boolean) [](start = 76, length = 51)

It doesn't look like changes in this file are related to the goal of the PR. At the very least I think they should be separated into a different PR. It also would be good to provide details about motivation for the change.

Reply:
Was not intended to be part of this pull request. Extracted to #40701.